DOCUMENTATION.md

```diff
@@ -432,14 +432,40 @@ The dataset source reads entities from a dataset in the data hub.
     "source": {
         "Type": "DatasetSource",
         "Name": "name of the dataset to read from",
-        "BatchSize" : "optional int value of how many to read at a time",
```

Comment:
removed because the correct place to set batchsize is at job level, not in the source.